### PR TITLE
Open main dialog gnome always

### PR DESCRIPTION
**Pull request by FlexW**
Fixes: https://github.com/nextcloud/desktop/issues/3564

### CODE REVIEW

File: src/gui/systray.h [hunk 1]
```diff
@@ -76,9 +76,9 @@ class Systray
     void openHelp();
     void shutdown();
 
-    Q_INVOKABLE void hideWindow();
```
Comment (mgallien) [inline src/gui/systray.h:79]: how did you check that they are not needed ?

Comment (FlexW) [reply src/gui/systray.h:79]: Those are signals. They are exposed automatically. https://doc.qt.io/qt-5/qtqml-cppintegration-exposecppattributes.html#exposing-signals

**Review by allexzander — APPROVED** [review]

File: src/gui/systray.h (hunk 1 above)
Comment (mgallien) [reply src/gui/systray.h:79]: I vaguely remember seeing those as warnings reported by clazy
I was asking to know if you had used a tool to do that

**Review by mgallien — APPROVED** [review]

File: src/gui/systray.h (hunk 1 above)
Comment (FlexW) [reply src/gui/systray.h:79]: I just saw them and then recalled that we talked about it. But yes I think too that clazy reports them.

Comment (FlexW) [reply src/gui/systray.h:79]: My idea was just to leave the code a bit cleaner than it was before I touched code in that area.

**Pull request merged**